src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -64,8 +77,41 @@
  * <p>Example:</p>
  * <pre>
  * abstract class AbstractFirstClass {} // OK
- * abstract class SecondClass {} // violation, it should match the pattern "^Abstract.+$"
+ * abstract class SecondClass {} // violation, it should match pattern "^Abstract.+$"
  * class AbstractThirdClass {} // OK, no "abstract" modifier
+ * class FourthClass {} // OK
+ * </pre>
+ * <p>
+ * How to configure the {@code AbstractClassName} to
+ * check names but ignore Name modifier:
+ * </p>
+ * <pre>
+ * &lt;module name="AbstractClassName"&gt;
+ *   &lt;property name="ignoreName" value="true"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * abstract class AbstractFirstClass {} // OK
+ * abstract class SecondClass {} // OK, ignored naming convention
+ * class AbstractThirdClass {} // violation, must be declared as 'abstract'
+ * class FourthClass {} // OK, not declared abstract
```

Comment:
Please update to `// OK, no "abstract" modifier`

Reply:
done